polling_stations/templates/fragments/cancelled_election.html

```diff
@@ -1,32 +1,22 @@
 {% load i18n_with_welsh %}
-{% load markdown_filter %}
```

Comment:
Uniquely, WDIV was actually set up to expect markdown. I've switched to to use `| linebreaks`